### ISSUE TITLE
Onset database

### ISSUE THREAD

**Issue by JamesBolt22** [CPJKU/onset_db#3]
Dear Mr Bock,

I have seen on a previous thread that you granted permission to access the database of audio and onsets on google drive. I am currently completing a masters degree in AI in music and would be grateful for access to this dataset?

Thank you for your time,

Kind regards,

James Bolt

Queen Mary University of London

**Comment by superbock** [comment]
Happy to share them with you.

**Comment by JamesBolt22** [comment]
Thank you for the share, I see I have access to the annotations. Is there access to the audio as well?

All the best,

James

**Comment by superbock** [comment]
I shared them with you as well :)

**Comment by JamesBolt22** [comment]
Thank you so much! One final technical question about your cnn onset detection paper if that is alright?

I was wondering how you choose the negative onsets. I understand that you slice the total spectrogram up into 15 frame wide chunks. If an onset lies in the middle of the chunk you class it as an onset, but does this not create many more negative examples than positives? If so how do you choose which negatives to keep to even the dataset?

I hope this makes sense,

Thank you for your time!

**Comment by superbock** [comment]
The spectrogram is sliced into *overlapping* chunks of length 15 frames, i.e. if the audio signal has N frames, there are also N chunks. A chunk is considered a positive example if the center frame contains an annotation for that frame. The neighboring frames are also considered positive examples, but we use a weight of 25% for these examples only.

**Comment by JamesBolt22** [comment]
Yes, that makes sense to me, So do all other chunks without the center frame (or frames to the left and right) count as negatives? If so does that not create significantly more negative examples than positive?

**Comment by superbock** [comment]
Yes, there are way more negative examples than positive ones. To compensate for this we use a lower threshold in the peak-picking stage.

**Comment by JamesBolt22** [comment]
Okay that makes a lot of sense! Thank you once again for your time. 

This has been very useful!

All the best.
